src/inverted-index.js

```diff
@@ -1,18 +1,25 @@
+/**
+ * Inverted index class
+ */
 class InvertedIndex {
-
+  /**
+   * Inverted index constructor
+   */
   constructor() {
-    //Object to hold the index
+    // Object to hold the index
     this.index = {};
   }
 
-  /** 
+  /**
    * @param{String} words - String to be tokonized
    * @return{Array} cleanContent
    */
   tokenize(words) {
```

Comment:
`tokenize()` is being used as a class method but is declared like an instance method.